### PR TITLE
Adding a yaml with the full (default) list of keyboard shortcuts

### PR DESCRIPTION
**Pull request by elviskahoro**

### CODE REVIEW

**Comment by elviskahoro** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4** <a href="https://app.graphite.dev/github/pr/warpdotdev/keysets/4" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/warpdotdev/keysets/4?utm_source=stack-comment).

File: default-warp-keybindings.yaml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+"pane_group:navigate_prev": cmd-[
+"pane_group:navigate_right": alt-cmd-right
+"pane_group:navigate_up": alt-cmd-up
+"pane_group:resize_down": ctrl-cmd-down
+"pane_group:resize_left": ctrl-cmd-left
+"pane_group:resize_right": ctrl-cmd-right
+"pane_group:resize_up": ctrl-cmd-up
+"pane_group:toggle_maximize_pane": shift-cmd-enter
+"terminal:bookmark_selected_block": cmd-b
+"terminal:copy": cmd-c
```
Comment (charlespierce) [inline default-warp-keybindings.yaml:66]: One that appears to be missing here:

```
"terminal:clear_lines": cmd-k
```

**Review by charlespierce — APPROVED** [review]
Looks good from what I can tell, with just a single binding that needs to be added!

**Review by MaggieShan — APPROVED** [review]
LGTM too!

File: default-warp-keybindings.yaml (hunk 1 above)
Comment (MaggieShan) [reply default-warp-keybindings.yaml:66]: Odd that I wasn't able to pull in **all** the keybindings though, will look into why that is 🧐

**Comment by MaggieShan** [comment]
Question: Some of the keybindings in this list are mutable and some immutable, by putting them all in this yaml file publicly are we hinting that users would be able to change all of them? Is that what we want?

**Comment by elviskahoro** [comment]
I feel like we shouldn't expose the immutable ones.

**Comment by elviskahoro** [comment]
> Question: Some of the keybindings in this list are mutable and some immutable, by putting them all in this yaml file publicly are we hinting that users would be able to change all of them? Is that what we want?

I feel like we shouldn't expose the immutable ones.

**Pull request merged**